client/blocks/jitm/index.jsx

```diff
@@ -122,23 +126,32 @@ function getEventHandlers( props, dispatch ) {
 	return handlers;
 }
 
-function useDevTool( { currentSite }, dispatch ) {
+function useDevTool( currentSite, dispatch ) {
```

Comment:
The `dispatch` function is used also by `getEventHandlers`, so instead of moving the `useDispatch()` call, I'm forced to do it twice. I decided not to do this change.